english/yuha/yulanda/yulanda.md

```diff
@@ -0,0 +1,3 @@
+Yulanda is your aunt from Boise, ID. She told you not to play this game but you did and now she's chasing after you to bring you back home.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.